assets/csidriveroperators/shared-resource/hypershift/mgmt/09_deployment.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: shared-resource-csi-driver-operator
+  namespace: ${CONTROLPLANE_NAMESPACE}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      name: shared-resource-csi-driver-operator
+  strategy: {}
+  template:
+    metadata:
+      annotations:
+        target.workload.openshift.io/management: '{"effect": "PreferredDuringScheduling"}'
+      labels:
+        name: shared-resource-csi-driver-operator
+    spec:
+      containers:
+      - args:
+        - start
+        - --kubeconfig=/etc/guest-kubeconfig/kubeconfig
+        env:
+        - name: DRIVER_IMAGE
+          value: ${DRIVER_IMAGE}
+        - name: NODE_DRIVER_REGISTRAR_IMAGE
+          value: ${NODE_DRIVER_REGISTRAR_IMAGE}
+        - name: WEBHOOK_IMAGE
+          value: ${WEBHOOK_IMAGE}
+        - name: HYPERSHIFT_IMAGE
+          value: ${HYPERSHIFT_IMAGE}
```

Comment:
Is this used anywhere in the operator? We added it to aws-ebs-csi-driver-operator, because it needs to mint cloud credentials, but here it's IMO useless.

Reply:
update to remove pushed